### PR TITLE
wzhzhzh

### PR DESCRIPTION
**Pull request by Ryll-Ryll**
## About The Pull Request
oranges said I could bring back wzhzhzh

## Why It's Good For The Game
improves immersions of coders who look in this file by helping them picture an operating microwave

### CODE REVIEW

**Comment by LemonInTheDark** [comment]
I want to speedmerge it

**Comment by Jack7D1** [comment]
cringe

File: code/modules/food_and_drinks/kitchen_machinery/microwave.dm [hunk 1]
```diff
@@ -255,7 +255,7 @@
 		break
```
Comment (Jack7D1) [inline code/modules/food_and_drinks/kitchen_machinery/microwave.dm:255]: Poor use of the break statement

**Review by Jack7D1 — APPROVED** [review]

File: code/modules/food_and_drinks/kitchen_machinery/microwave.dm (hunk 1 above)
Comment (LemonInTheDark) [reply code/modules/food_and_drinks/kitchen_machinery/microwave.dm:255]: I mean it's a little silly, but it's ok. the min doesn't need to be there, and the fact it is makes me think it's aged 20mmmmm13? code?
image

Comment (Jack7D1) [reply code/modules/food_and_drinks/kitchen_machinery/microwave.dm:255]: the fact ryll didn't fix this speaks a lot to our society

Comment (LemonInTheDark) [reply code/modules/food_and_drinks/kitchen_machinery/microwave.dm:255]: it's also not even their code, so even more EH

**Comment by Couls** [comment]
i hate this

**Comment by Jared-Fogle** [comment]
wzhzhzh

**Review by Fikou — APPROVED** [review]

**Comment by manray0** [comment]
yes hell yes

**Pull request merged**